Feature/sp 158 (#517)
appcircleio/appcircle-docusaurus: docs/marketplace/fastlane/index.md

---
title: Fastlane Marketplace
description: Appcircle fastlane plugin allows you to distribute your builds to testers directly pipeline.
tags:
  [
    github-marketplace,
    testing-distribution,
    ipa distribution,
    apk distribution,
    binary distribution,
  ]
sidebar_position: 2
---

> Enhance power of distributing the builds to test groups or release beta version of your apps with using **Appcircle Testing Distribution** and **Appcircle Enterprise Store** plugins

Reviewer: Enhance the power of distributing builds to test groups or releasing beta versions of your apps by using the **Appcircle Testing Distribution** and **Appcircle Enterprise Store** plugins.

> Enhance power of distributing the builds to test groups or release beta version of your apps with using **Appcircle Testing Distribution** and **Appcircle Enterprise Store** plugins

Reviewer: ```suggestion
Enhance the power of distributing builds to test groups or releasing beta versions of your apps by using the **Appcircle Testing Distribution** and **Appcircle Enterprise Store** plugins.
```

### Testing Distribution

> It is a process distributing test builds to designated test groups or individuals.

Reviewer: It is a process of distributing test builds to designated test groups or individuals.

> It is a process distributing test builds to designated test groups or individuals.

Reviewer: ```suggestion
It is a process of distributing test builds to designated test groups or individuals.```

> It is a process distributing test builds to designated test groups or individuals.

Reviewer: ```suggestion
It is a process of distributing test builds to designated test groups or individuals.
```
